docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -177,82 +183,82 @@ Due to the Zeebe data replication, no data has been lost.
 
 #### Desired state
 
-You are creating a temporary Camunda 8 deployment within the same region, but different namespace, to recover the Zeebe cluster functionality. Using a different namespace allows for easier distinguishing between the normal Zeebe deployment and Zeebe failover deployment.
-
-The newly deployed Zeebe brokers will be running in the failover mode. This will restore the quorum and the Zeebe data processing. Additionally, the new failover brokers are configured to export the data to the surviving Elasticsearch instance and to the newly deployed failover Elasticsearch instance.
+You have removed the lost brokers from the Zeebe cluster. This will allow us to continue processing after the next step and ensure that the new brokers in the failback procedure will only join the cluster with our intervention.
 
 #### How to get there
 
-In the case **Region 1** was lost: in the previously cloned repository [c8-multi-region](https://github.com/camunda/c8-multi-region), navigate to the folder [aws/dual-region/kubernetes/region0](https://github.com/camunda/c8-multi-region/blob/main/aws/dual-region/kubernetes/region0/). This contains the example Helm values yaml `camunda-values-failover.yml` containing the required overlay for the **failover** mode.
+You will port-forward the `Zeebe Gateway` in the surviving region to the local host to interact with the Gateway.
 
-In the case when your **Region 0** was lost, instead go to the folder [aws/dual-region/kubernetes/region1](https://github.com/camunda/c8-multi-region/blob/main/aws/dual-region/kubernetes/region1/) for the `camunda-values-failover.yml` file.
+The following alternatives to port-forwarding are possible:
 
-The chosen `camunda-values-failover.yml` requires adjustments before installing the Helm chart and the same has to be done for the base `camunda-values.yml` in `aws/dual-region/kubernetes`.
+- if it's exposed to the outside, one can skip port-forwarding and use the URL directly
+- one can use, for example, the Elasticsearch pod to exec into it and curl from there
```

Comment:
```suggestion
- one can `exec` into an existing pod (such as Elasticsearch), and `curl` from there
```

^ Is this a correct rephrasing? Some clarity + command backtics.

Reply:
sure, I also linked to the official k8s exec docs on the command.